easybuild/easyconfigs/g/gffcompare/gffcompare-0.10.6-GCCcore-7.3.0.eb

```
@@ -0,0 +1,33 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'MakeCp'
+
+name = 'gffcompare'
+version = '0.10.6'
+
+homepage = 'https://github.com/gpertea/%(namelower)s'
+description = """gffcompare provides classification and reference annotation mapping and
+ matching statistics for RNA-Seq assemblies (transfrags) or other generic GFF/GTF files."""
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+
+source_urls = ['http://ccb.jhu.edu/software/stringtie/dl/']
+sources = ['%(namelower)s-%(version)s.tar.gz']
```

Use SOURCE_TAR_GZ instead of %(namelower)s-%(version)s.tar.gz

---

With the name change above:

```suggestion
sources = [SOURCELOWER_TAR_GZ]
```